Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/BooleanSetting.java

```diff
@@ -237,7 +239,7 @@
           MAIN_ENABLE_CHEATS,
           MAIN_OVERRIDE_REGION_SETTINGS,
           MAIN_MMU,
-          MAIN_DSP_JIT,
+          MAIN_DSP_JIT
```

Comment:
This change should be undone. We prefer to have a comma at the end, for the sake of making diffs smaller.